### PR TITLE
Adding eventing v1.9

### PR DESCRIPTION
**Pull request by creydr**
Signed-off-by: Christoph Stäbler <cstabler@redhat.com>

### CODE REVIEW

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-images
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-conformance-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-e2e-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-reconciler-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-images
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-conformance-aws-ocp-48
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-e2e-aws-ocp-48
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-reconciler-aws-ocp-48

File: core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay [hunk 1]
```diff
@@ -1,22 +0,0 @@
-registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher quay.io/openshift-knative/knative-eventing-kafka-broker-dispatcher
```
Comment (pierDipi) [inline core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay:1]: There is a bug in the generator that ends up dropping this files that are needed

Comment (creydr) [reply core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay]: Ah ok. I was wondering, but didn't remember related issues last time... Updated it...

**Comment by creydr** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-images
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-conformance-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-e2e-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-412-test-reconciler-aws-ocp-412
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-images
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-conformance-aws-ocp-48
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-e2e-aws-ocp-48
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.9-48-test-reconciler-aws-ocp-48

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**